sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -183,7 +183,7 @@ void AddNonNullPolicies(HttpPipelinePolicy[] policiesToAdd)
             {
                 if (buildOptions.ClientOptions.IsCustomTransportSet)
                 {
-                    if (AzureCoreEventSource.Singleton.IsEnabled())
+                    if (AzureCoreEventSource.Singleton.IsEnabledInformational())
```

Comment:
Curious: how do we know that this doesn't change the behavior from what it was previously?  Was there a guarantee that this was only logged at the informational level?

Reply:
Each event method is marked with `EventAttribute` which specifies event level. So these added checks match those levels.